### PR TITLE
NAS-108070 / 20.12 / Nas 108070 107881 108069

### PR DESCRIPTION
**Pull request by lola27**

### CODE REVIEW

File: src/app/helptext/task-calendar/replication/replication-wizard.ts [hunk 1]
```diff
@@ -54,25 +54,25 @@ export default {
  the selected dataset snapshots.'),
 
     encryption_placeholder: T('Encryption'),
-    encryption_tooltip: T(''),
+    encryption_tooltip: T('Checking this box, enables additional options for encryption replication'),
```
Comment (Mrt134) [inline src/app/helptext/task-calendar/replication/replication-wizard.ts:57]: Cleanup suggestion:
"Set to use encryption when replicating data. Additional encryption options will appear."

File: src/app/helptext/task-calendar/replication/replication-wizard.ts [hunk 2]
```diff
@@ -54,25 +54,25 @@ export default {
...
 
     encryption_key_generate_placeholder: T('Generate Encryption Key'),
-    encryption_key_generate_tooltip: T(''),
+    encryption_key_generate_tooltip: T('If Hex key type is selected, this will autogenerate an encryption key'),
```
Comment (Mrt134) [inline src/app/helptext/task-calendar/replication/replication-wizard.ts:60]: Cleanup suggestion:
"If the <i>Hex key</i> type is chosen, an encryption key will be autogenerated."

File: src/app/helptext/task-calendar/replication/replication-wizard.ts [hunk 3]
```diff
@@ -54,25 +54,25 @@ export default {
...
 
     encryption_key_hex_placeholder: T('Encryption Key'),
-    encryption_key_hex_tooltip: T(''),
+    encryption_key_hex_tooltip: T('Uncheck Generate Encryption Key to import your custom Hex key'),
```
Comment (Mrt134) [inline src/app/helptext/task-calendar/replication/replication-wizard.ts:63]: Cleanup suggestion:
"Unset <i>Generate Encryption Key</i> to instead import a custom Hex key."

File: src/app/helptext/task-calendar/replication/replication-wizard.ts [hunk 4]
```diff
@@ -54,25 +54,25 @@ export default {
...
 
     encryption_key_passphrase_placeholder: T('Passphrase'),
-    encryption_key_passphrase_tooltip: T(''),
+    encryption_key_passphrase_tooltip: T('If Passphrase key type is selected, enter an alphanumeric encryption key'),
```
Comment (Mrt134) [inline src/app/helptext/task-calendar/replication/replication-wizard.ts:66]: Cleanup suggestion:
"Enter an alphanumeric encryption key. Only available when <i>Passphrase</i> is the chosen key format.

File: src/app/helptext/task-calendar/replication/replication-wizard.ts [hunk 5]
```diff
@@ -54,25 +54,25 @@ export default {
...
 
     encryption_key_format_placeholder: T('Encryption Key Format'),
-    encryption_key_format_tooltip: T(''),
+    encryption_key_format_tooltip: T('Select Hex or Passphrase for encryption key type'),
 
     encryption_key_location_truenasdb_placeholder: T('Store Encryption key in Sending TrueNAS database'),
-    encryption_key_location_truenasdb_tooltip: T(''),
+    encryption_key_location_truenasdb_tooltip: T('Check this box to store the encryption key in the TrueNAS Database'),
```
Comment (Mrt134) [inline src/app/helptext/task-calendar/replication/replication-wizard.ts:72]: Cleanup suggestion:
"Set to store the encryption key in the TrueNAS database."

File: src/app/helptext/task-calendar/replication/replication-wizard.ts [hunk 6]
```diff
@@ -54,25 +54,25 @@ export default {
...
 
     encryption_key_location_placeholder: T('Encryption Key Location in Target System'),
-    encryption_key_location_tooltip: T(''),
+    encryption_key_location_tooltip: T('Temporary location for encryption key used during ZFS receive'),
```
Comment (Mrt134) [inline src/app/helptext/task-calendar/replication/replication-wizard.ts:75]: Cleanup suggestion:
"Choose a temporary location to store the encryption key that decrypts the replicated data."

File: src/app/helptext/task-calendar/replication/replication.ts [hunk 7]
```diff
@@ -191,25 +191,25 @@ from the source dataset.'),
  <code>readonly</code> property during replication.'),
```
Comment (Mrt134) [inline src/app/helptext/task-calendar/replication/replication.ts:192]: Sync up any changes from the wizard help text into this file, or rework the strings to call help text from a single source file.

**Review by Mrt134 — COMMENTED** [review]
Left a few comments for text cleanups.
One other suggestion is that if the help text string is supposed to be identical between the wizard and the advanced form, set up a call to import the text into one of the forms. That way there is only a single source of text that can't drift apart and cause confusion.

**Review by erinix — APPROVED** [review]
lgtm

**Pull request merged**